packages/angular_devkit/schematics/tasks/tslint-fix/executor.ts

```diff
@@ -182,7 +182,10 @@ export default function(): TaskExecutor<TslintFixTaskOptions> {
         }
         const formatter = new Formatter();
 
-        const output = formatter.format(result.failures, result.fixes);
+        // Certain tslint formatters outputs '\n' when there are no failures.
+        // This will bloat the console when having schematics running refactor tasks.
+        // see https://github.com/palantir/tslint/issues/4244
+        const output = formatter.format(result.failures, result.fixes).trim();
```

Comment:
By looking at the type definition for formatters, they should always return a `string`. But not sure if it would be worth having a safety check.

* https://github.com/palantir/tslint/blob/master/src/language/formatter/abstractFormatter.ts#L23

Reply:
Yeah they never return undefined.

Reply:
They can be specified directly as a file path and their operation is completely out of our control.   One extra safety check goes along way to improved user experience.  Assume external code will fail.

Reply:
Add additional check